### ISSUE TITLE
No JEI recipe for Black Iron Ingots

### ISSUE THREAD

**Issue by dgobi** [Freebi85/Dungeons-Dragons-Space-Shuttles#323]
Hello,

JEI currently does not show the correct recipe for Black Iron Ingots. It shows:

-  9 nuggets = 1 ingot
- 9 ingots = 1 block (and vice versa)
- Factorizer recipes for same

What is missing is the actual recipe, which is black dye (I assume this is oredict but I made mine with squid ink) and an iron ingot.

Thanks for the great pack. Most fun I've had in modded MC since Blood n Bones!

**Comment by Freebi85** [comment]
Thank you im glad you like it =)

But the recipe is there. Maybe forgot to switch to next page?^^

![2019-09-06_19 33 23](https://user-images.githubusercontent.com/42634622/64448148-4e35a880-d0dd-11e9-8770-cfacf913c2d0.png)

**Comment by dgobi** [comment]
Ok that's really strange! It was genuinely not showing the black dye recipe earlier, but maybe that has to do with the path I took to get to JEI? Maybe because I accessed it from within a table that had access to an inventory or something. FWIW I saw a similar report in the comments for Extended Crafting, so maybe something specific to the mod, too.

Anyways, I can also clearly see it now in my JEI so I can't explain, and this is effectively a non-issue. Sorry for the bother!
